app/src/main/res/layout/item_song.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
You should read issue description more attentively: 
Because for now we need only Image,Title, Artist and Song duration - so remove Like and Menu icons